crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -113,7 +113,7 @@ def compute_status
           )
         end
         nova = NodeObject.find("roles:nova-controller").first
-        ret[:no_live_migration] = true unless nova["nova"]["use_migration"]
```

Comment:
After inverting the logic it doesn't look bad imo. And we got rid of `unless` :)